conf/nginx.conf

```diff
@@ -0,0 +1,16 @@
+server {
+    listen       80;
+    server_name  _;
+
+
+    location /sitestatic/ {
+        alias /var/www/static/;
+    }
+
+    location / {
+      proxy_pass http://rapidpro:5000;
```

Comment:
We should use `uwsgi_pass` here, if only to be consistent with the rest of our projects (the uWSGI protocal is negligibly faster than HTTP - but we don't really care about that).